wolfssl/openssl/dsa.h

```diff
@@ -4,13 +4,17 @@
 #ifndef WOLFSSL_DSA_H_
 #define WOLFSSL_DSA_H_
 
-#include <wolfssl/openssl/ssl.h>
```

Comment:
Removing this include affects compatibility. An example is a test file that just includes openssl/dsa.h an no other files. Linked to OpenSSL this test would compile fine while declaring DSA* dsa;. With this removed though it would not compile linked to wolfSSL compatibility layer. Same for ec.h.